### DISCUSSION TITLE
How to support wikitext?

### DISCUSSION THREAD

**Discussion by linonetwo** [micromark/micromark#97] [Q&A] [answered]
I'm using https://github.com/tiddly-gittly/TiddlyGit-Desktop , it supports both markdown and wikitext (inwhich we can use macro and wiki widget...), I hope I can extend the parser to parse wikitext, so I can use micromark to parse them all together.

I saw https://github.com/micromark/micromark/discussions/83 said it is easy to extend the syntax ( so I can add `[[ ]]` link format), so is it possible to disable syntax? (So I can disable `[]()` syntax in wikitext mode)

**Comment by wooorm** [comment, marked as answer]
I don’t think anyone every said it’s “easy” to extend the syntax.

I believe the info you’re looking for is extensively documented in the readme under [§ Extensions](https://github.com/micromark/micromark#extensions)

**Reply by linonetwo** [reply]
Ok, that means, by delete a syntax plugin from the plugin list, we can disable a builtin markdown syntax?

**Reply by wooorm** [reply]
in this case I think you want to turn off label end: https://github.com/micromark/micromark/blob/fa21fccd3cc53285380b5cae13dfe546483ecb6f/packages/micromark-core-commonmark/dev/lib/label-end.js#L26.
You can do that with an extension.

**Comment by ChristianMurphy** [comment]
For wiki-links specifically, there is an existing plugin which may be able to help https://github.com/landakram/remark-wiki-link
